### PR TITLE
Feature/reducers actions

### PR DESCRIPTION
**Pull request by CesarHerr**
- Add Redux Toolkit. :heavy_check_mark: 
- Create a directory that will contain all your Redux logic (/src/redux). :heavy_check_mark: 
- Configure a Redux store (/src/redux/store.js). :heavy_check_mark: 

Define a slice of state for books that:
- Includes an array of books (initial state: empty array) :heavy_check_mark: 
- Includes a reducer that adds a book. :heavy_check_mark: 
- Includes a reducer that removes a book. :heavy_check_mark: 

Define a slice of state for categories that:
- Includes an array of categories (initial state: empty array). :heavy_check_mark: 
- Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string). :heavy_check_mark: 
- Structure your application files using a "feature folder" approach and use the ducks pattern for your Redux files. :heavy_check_mark:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,7 +168,7 @@ You can deploy this project using:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- Add reducers and actions.
+
```
Comment (rustam-tolipov) [inline README.md:171]: - [ ] You did a great job! The only thing we need to fix it your `issues` link. Kindly fix the link and resubmit to get approved.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @CesarHerr,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well implemented code_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (CesarHerr) [reply README.md:171]: @Rustamxon7  I already did it, thanks for your review! :handshake:

**Review by DuaneDave — APPROVED** [review]
Hi @CesarHerr 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Your readme is very professional
✔️ Reducers are set up correctly

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CesarHerr** [comment]
@DuaneDave Thanks for your review!! :smile:

**Pull request merged**